go-gorilla/templates/timeline.html

```diff
@@ -1,86 +1,83 @@
+          {{end}}
       </div>
+    {{else if eq .Endpoint "public_timeline"}}
+      <h2>Public Timeline</h2>
+    {{else if eq .Endpoint "my_timeline"}}
+    <h2>My Timeline</h2>
+    <div class="twitbox">
+      <h3>What's on your mind, {{.User}}?</h3>
+      <form action="{{url_for "add_message" "" }}" method="POST">
+        <p>
+          <input type="text" name="text" size="60">
+          <input type="submit" value="Share">
+        </p>
+      </form>
+    </div>
     {{end}}
 
-    <div>
-      {{if ne .Req (print "/public")}}
-        </div>
-        {{if eq .User .ProfileUser}}
-          <div class="twitbox">
-            <h3>What's on your mind, {{.User}}?</h3>
-            <form action="{{url_for "add_message" ""}}" method="POST">
-              <p>
-                <input type="text" name="text" size="60">
-                <input type="submit" value="Share">
-              </p>
-            </form>
-          </div>
-        {{end}}
-        
+    {{if .FlashMessages}}
+    <div class="flashes">
+      {{range .FlashMessages}}
+      <div>{{.}}</div>
       {{end}}
-      <ul class="messages">
-        {{range $fields := .Messages}}
-          <li>
-            <img src="{{getavatar $fields.Email 48}}" alt="{{$fields.Username}}'s avatar">
-            <strong><a href="{{url_for "timeline" (formatUsernameUrl $fields.Username)}}">{{$fields.Username}}</a></strong>
-            {{$fields.Text}}
-            <small>&mdash; {{gettimestamp $fields.PubDate}}</small>
-      
-            {{if $user}} <!-- Check if user is logged in -->
-              <div class="followstatus">
-                {{if eq $user $fields.Username}}
-                  This is you!
-                {{else if IsFollowing $following $fields.AuthorID}}
-                  You are currently following this user
-                  <a class="unfollow" href="{{url_for "unfollow" (formatUsernameUrl $fields.Username)}}">Unfollow user</a>
-                {{else}}
-                  You are not yet following this user
-                  <a class="follow" href="{{url_for "follow" (formatUsernameUrl $fields.Username)}}">Follow user</a>
-                {{end}}
-              </div>
-            {{end}}
-          </li>
-        {{end}}
-      </ul>
     </div>
-    </div>
-    
-
-
+    {{end}}
+    <ul class="messages">
+      {{if .Messages}}
+      {{range $fields := .Messages}}
+      <li>
+        <img src="{{getavatar $fields.Email 48}}" alt="{{$fields.Username}}'s avatar">
+        <strong><a href="{{url_for "timeline" (formatUsernameUrl $fields.Username)}}">{{$fields.Username}}</a></strong>
+        {{$fields.Text}}
+        <small>&mdash; {{gettimestamp $fields.PubDate}}</small>
+      </li>
+      {{end}}
+      {{else}}
+      <li><em>There's no message so far.</em></li>
+      {{end}}
+    </ul>
   <div class="footer">
-    Minitwit - Go application
+   Minitwit Go-gorilla 1.23.1
```

Comment:
We agreed on: 
`MiniTwit python-flask Application` so in your case:
`MiniTwit go-gorilla Application`

Reply:
ok